test/lib/postgres.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+const assert = require('assert').strict;
+const {PostgresDatabase} = require('../../lib');
+
+function testMod(mod) {
+	try {
+		require(mod);
+	} catch (e) {
+		return it.skip;
+	}
+	return it;
+}
+
+// only run these if you already have postgres configured
+describe.skip("Postgres features", () => {
+	it("Should be able to connect to a database", async () => {
+		this.database = new PostgresDatabase();
+	});
+	it("Should be able to insert data", async () => {
+		assert.doesNotThrow(async () => {
```

Comment:
`doesNotThrow` doesn't work with Promises IIRC.
```suggestion
		await assert.doesNotThrowAsync(async () => {
```